Fail to build with gcc10 with "multiple definition of `attack_count'"

Original Debian bug report:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=957524

Relevant build log:
```
/usr/bin/ld: channelhopper.o:./src/./attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/attacks.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/auth_dos.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/beacon_flood.o:./src/attacks/beacon_flood.c:34: multiple definition of `bssid'; attacks/auth_dos.o:./src/attacks/auth_dos.c:50: first defined here
/usr/bin/ld: attacks/beacon_flood.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/countermeasures.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/deauth.o:./src/attacks/deauth.c:19: multiple definition of `bssid'; attacks/auth_dos.o:./src/attacks/auth_dos.c:50: first defined here
/usr/bin/ld: attacks/deauth.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/dummy.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/eapol.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/fuzzer.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/ieee80211s.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/probing.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
/usr/bin/ld: attacks/wids.o:./src/attacks/attacks.h:39: multiple definition of `attack_count'; /tmp/cc8aVa1n.o:./src/attacks/attacks.h:39: first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:36: mdk4] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>/src'
make[1]: *** [Makefile:10: all] Error 2
```

Reference for the issue:
https://gcc.gnu.org/gcc-10/porting_to.html#common

--- a/src/Makefile
+++ b/src/Makefile
@@ -1,7 +1,9 @@
 MDK_ROOT = ..
 include $(MDK_ROOT)/common.mak
 
-CFLAGS		+= -g -O3 -Wall -Wextra 
+# TODO: Remove '-fcommon' from CFLAGS and address issues, reference:
+# https://gcc.gnu.org/gcc-10/porting_to.html#common
+CFLAGS		+= -g -O3 -Wall -Wextra -fcommon
 LINKFLAGS	= -lpthread -lpcap $(LDFLAGS)
 
 SBINDIR		= $(PREFIX)/sbin

